opcua/client/ua_client.py

```diff
@@ -477,3 +477,14 @@ def history_read(self, params):
         self.logger.debug(response)
         response.ResponseHeader.ServiceResult.check()
         return response.Results
+
+
+def modify_monitored_item_low(self, params):
```

Comment:
Yes, that was the point. I was writing everything in the same file and named it like that to differentiate them easier. Will correct it.